### PR TITLE
Add endpoint for disability rating

### PR DESCRIPTION
**Pull request by austin-martinez**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -24,6 +24,7 @@ before_script:
   - export MHV_SM_USER_ID='10616687'
   - export EVSS_CLAIMS_BASE_URL='http://csraciapp6.evss.srarad.com:7003/wss-claims-services-web-3.1/rest'
   - export EVSS_DOCUMENTS_BASE_URL='http://csraciapp6.evss.srarad.com:7003/wss-document-services-web-3.1/rest/'
+  - export EVSS_COMMON_BASE_URL='http://csraciapp8.evss.srarad.com:7003/wss-common-services-web-11.1/rest/'
```
Comment (carlmjohnson) [inline .travis.yml]: We're supposed to document these in the README and notify devops.

File: app/controllers/v0/disability_ratings_controller.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+module V0
+  class DisabilityRatingsController < ApplicationController
+    skip_before_action :authenticate
+
+    def show
+      disability_rating = DisabilityRating.new(current_user.rating_record['disabilityRatingRecord'])
```
Comment (carlmjohnson) [inline app/controllers/v0/disability_ratings_controller.rb]: Can the `['disabilityRatingRecord']` part go into the serializer?

File: app/serializers/disability_rating_serializer.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+class DisabilityRatingSerializer < ActiveModel::Serializer
+  attributes :ratings, :service_connected_combined_degree
+
+  # activemodel serializer requires an id attr for json-api spec
+  def id
+    0
```
Comment (carlmjohnson) [inline app/serializers/disability_rating_serializer.rb]: Can we return `nil` here or will Active Model complain?

File: app/controllers/v0/disability_ratings_controller.rb (hunk 2 above)
Comment (austin-martinez) [inline app/controllers/v0/disability_ratings_controller.rb]: The disability rating object/serializer shouldn't need to know how to access the disability rating information from the rating record, they just get passed the data that is applicable. We could just have an endpoint for returning the whole rating record but I didn't see any use cases for the other data at this time

File: lib/evss/claims_service.rb [hunk 4]
```diff
@@ -25,7 +25,7 @@ def submit_5103_waiver(claim_id)
     protected
 
     def base_url
-      ENV['EVSS_CLAIMS_BASE_URL']
+      "#{ENV['EVSS_HOST']}/wss-claims-services-web-3.1/rest"
```
Comment (carlmjohnson) [inline lib/evss/claims_service.rb]: Smart.

**Pull request merged**